.editorconfig

```diff
@@ -0,0 +1,8 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
All the IDE are not natively compatible with that, and for VSCode it seems (from their website) that you'll need a plugin.

Reply:
It's optional configuration, so that shouldn't be a problem.